### PR TITLE
saving bus stop id into dynamo on BusByRouteIntent

### PR DESCRIPTION
**Pull request by goody**

### CODE REVIEW

**Comment by goody** [comment]
@BoldBigflank what do you think of this?

File: alexa.js [hunk 1]
```diff
@@ -195,14 +195,23 @@ function getBusByRoute(intent, session, callback) {
             shouldEndSession = results.isError ? false : true;
             repromptText = results.repromptText ? results.repromptText : '';
             if(!results.isError) {
+                var slots = {
```
Comment (BoldBigflank) [inline alexa.js:198]: Don't need to define slots, it's in `intent.slots`

**Comment by BoldBigflank** [comment]
I see what you're doing, since the getBusByRoute function ends up calling getStopSchedule the same as if you told it a stop number. A future improvement would be to only speak buses that match the route, so I think we should keep the original intent.slots and make getLastBus call getBusByRoute when it has the extra info. This is ready to do when you set the busType to "route".

**Comment by goody** [comment]
Ah...got it.

**Pull request closed without merging**